limit pivotMotor angle from 40 to 90 degrees

awfulcarpet opened the pull request:
limit pivotMotor from turning too much

awfulcarpet committed:
limit shooterMotor angle from 40 to 90 degrees

awfulcarpet committed:
update limits to be in rotations rather than degrees

Jnesselr commented on src/main/java/frc/robot/subsystems/Shooter.java:
```diff
@@ -70,7 +71,13 @@ public void shooterInit() {
-    // TODO apply SoftwareLimitSwitchConfigs to pivotMotor to prevent the robot arm from crashing
+
+    SoftwareLimitSwitchConfigs pivotLimits = new SoftwareLimitSwitchConfigs();
+    pivotLimits.ForwardSoftLimitThreshold = 50.0 / 360.0; // maximum of 90 degrees
+    pivotLimits.ReverseSoftLimitThreshold = 0;
```
I don't think this is right. First things first, the limit is actually a minimum of like 30 ish and I set the setpoint to just below 30 so it doesn't crash into anything when it's dropping down. And second of all, I wouldn't think that you would need to offset by the minimum to get the rotations value since the wrist calibration is accommodates the zero point correctly and knows setting it at an angle should be ~30 degrees.

awfulcarpet committed:
fix min and max threshold for software limit switches to not offset by
encoder offset

change min to 30 degrees as it is slightly above where the arm can be a
t its lowest point

change max to 90 degrees, should't have an effect when we are going from
starting position in the match to default position
